EventSubscriber/MenuSubscriber.php

```diff
@@ -50,87 +33,22 @@ public static function getSubscribedEvents(): array
 
     public function onMenuConfigure(ConfigureMainMenuEvent $event): void
     {
-        $currentUser = $this->getUser();
+        $currentUser = $this->security->getUser();
         if ($currentUser === null) {
             return;
         }
 
-        $users = $this->getUsers();
-        $dataToMenuItem = $this->approvalRepository->findCurrentWeekToApprove($users, $currentUser);
-
-        $date = date('Y-m-d', strtotime('this week'));
+        $model = new MenuItemModel(
+            'approvalBundle', 'title.approval_bundle', 'approval_bundle_report', [], 'fas fa-thumbs-up',
+        );
 
-        $isTeamLeadOrAdmin = $this->security->isGranted('view_all_approval') || $this->security->isGranted('view_team_approval');
-        $menu = $event->getMenu();
-        if (empty($users)) {
-            $menu->addItem(
-                new MenuItemModel(
-                    'approvalBundle',
-                    'title.approval_bundle',
-                    'approval_bundle_settings',
-                    [],
-                    'fas fa-thumbs-up'
-                )
-            );
-        } else {
-            $menu->addItem(
```

Comment:
Can it be that the V2 does not support the small number next to the menu item? If so, then this can be removed totally from the source? Or is there a plan to support those numbers somewhen?
![image](https://github.com/KatjaGlassConsulting/ApprovalBundle/assets/55694577/85d95338-17dd-44ed-a825-80087ae06478)

Reply:
Hm, good finding. Right now, badges are only supported on the second menu level.

As said before, I would vote for removing this entirely, as it a) causes complex database queries and b) is not used anywhere else in Kimai (only MR code used it).

But I could offer an integration into the notification dropdown instead:
<img width="401" alt="Bildschirmfoto 2023-12-27 um 12 23 34" src="https://github.com/KatjaGlassConsulting/ApprovalBundle/assets/533162/ec81cb6a-a07a-4f5a-a36c-3d2a99805a87">

We should think about caching the info though in the future.

Reply:
Let's remove it in V2 as you did.